### DISCUSSION TITLE
HOW APSIM CALCULATES NITROGEN MINERALIZATION

### DISCUSSION THREAD

**Discussion by Malitoz** [APSIMInitiative/ApsimX#9001] [Q&A]
What is the formular used to calculate Nmin in APSIMNextGet?

**Comment by hut104** [comment]
The algorithms are essentially still reproducing the original model of Probert et al
https://www.sciencedirect.com/science/article/pii/S0308521X97000280
[https://ars.els-cdn.com/content/image/1-s2.0-S0308521X24X00037-cov150h.gif]<https://www.sciencedirect.com/science/article/pii/S0308521X97000280>
APSIM's water and nitrogen modules and simulation of the dynamics of water and nitrogen in fallow systems<https://www.sciencedirect.com/science/article/pii/S0308521X97000280>
APSIM (Agricultural Production Systems Simulator) is a software system which provides a flexible structure for the simulation of climatic and soil man…
www.sciencedirect.com



________________________________
From: Lucas Mutuma ***@***.***>
Sent: Monday, 3 June 2024 10:59 PM
To: APSIMInitiative/ApsimX ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [APSIMInitiative/ApsimX] HOW APSIM CALCULATES NITROGEN MINERALIZATION (Discussion #9001)


What is the formular used to calculate Nmin in APSIMNextGet?
